src/snowflake/snowpark/modin/pandas/series_utils.py

```diff
@@ -1428,90 +1430,82 @@ def tz(self) -> "tzinfo | None":
+        return Series(query_compiler=self._query_compiler.dt_property("floor"))
 
     def ceil(self, *args, **kwargs):
-        return Series(query_compiler=self._query_compiler.dt_ceil(*args, **kwargs))
+        return Series(query_compiler=self._query_compiler.dt_property("ceil"))
 
     def month_name(self, *args, **kwargs):
-        return Series(
-            query_compiler=self._query_compiler.dt_month_name(*args, **kwargs)
-        )
+        return Series(query_compiler=self._query_compiler.dt_property("month_name"))
 
     def day_name(self, *args, **kwargs):
-        return Series(query_compiler=self._query_compiler.dt_day_name(*args, **kwargs))
+        return Series(query_compiler=self._query_compiler.dt_property("day_name"))
 
     def total_seconds(self, *args, **kwargs):
-        return Series(
-            query_compiler=self._query_compiler.dt_total_seconds(*args, **kwargs)
-        )
+        return Series(query_compiler=self._query_compiler.dt_property("total_seconds"))
 
     def to_pytimedelta(self) -> "npt.NDArray[np.object_]":
-        res = self._query_compiler.dt_to_pytimedelta()
+        res = self._query_compiler.dt_property("to_pytimedelta")
         return res.to_numpy()[:, 0]
 
     @property
     def seconds(self):
-        return Series(query_compiler=self._query_compiler.dt_seconds())
+        return Series(query_compiler=self._query_compiler.dt_property("seconds"))
 
     @property
     def days(self):
-        return Series(query_compiler=self._query_compiler.dt_days())
+        return Series(query_compiler=self._query_compiler.dt_property("days"))
 
     @property
     def microseconds(self):
-        return Series(query_compiler=self._query_compiler.dt_microseconds())
+        return Series(query_compiler=self._query_compiler.dt_property("microseconds"))
 
     @property
     def nanoseconds(self):
-        return Series(query_compiler=self._query_compiler.dt_nanoseconds())
+        return Series(query_compiler=self._query_compiler.dt_property("nanoseconds"))
 
     @property
     def components(self):
 
-        return DataFrame(query_compiler=self._query_compiler.dt_components())
+        return DataFrame(query_compiler=self._query_compiler.dt_property("components"))
 
     @property
     def qyear(self):
-        return Series(query_compiler=self._query_compiler.dt_qyear())
+        return Series(query_compiler=self._query_compiler.dt_property("qyear"))
 
     @property
     def start_time(self):
-        return Series(query_compiler=self._query_compiler.dt_start_time())
+        return Series(query_compiler=self._query_compiler.dt_property("start_time"))
 
     @property
     def end_time(self):
-        return Series(query_compiler=self._query_compiler.dt_end_time())
+        return Series(query_compiler=self._query_compiler.dt_property("end_time"))
 
     def to_timestamp(self, *args, **kwargs):
-        return Series(
-            query_compiler=self._query_compiler.dt_to_timestamp(*args, **kwargs)
```

Comment:
`Series.dt.to_timestamp` doesn't exist in pandas docs. But I can see it in Modin OSS. Not sure if it's intentional or a mistake in Modin. @sfc-gh-mvashishtha any idea?

Reply:
that looks like an oversight in Modin: https://github.com/modin-project/modin/issues/7232